BlazorApp1/wwwroot/schema.json

```diff
@@ -0,0 +1,8 @@
+[{
```

Comment:
please rename it with chart name it is connected to

Reply:
done.